core/capabilities/launcher.go

```diff
@@ -398,7 +398,8 @@ func (w *launcher) addToRegistryAndSetDispatcher(ctx context.Context, capability
 }
 
 var (
-	defaultTargetRequestTimeout = time.Minute
+	// TODO: make this configurable
+	defaultTargetRequestTimeout = 10 * time.Minute
```

Comment:
what does this mean?

if we are writing reports on aptos at up to once every 5 seconds, how can there be a 10 minute timeout in system?

Reply:
added PR description